### ISSUE TITLE
Error loading boost

### ISSUE THREAD

**Issue by hadim** [conda-forge/qvina-feedstock#1]
Here is the error using the package on Linux:

```
qvina2: error while loading shared libraries: libboost_filesystem.so.1.72.0: cannot open shared object file: No such file or directory
```

The package has been built against 1.72 but the recipe says:

```yaml
requirements:
  build:
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
    - cmake >=3.12
    - make
  host:
    - boost-cpp >=1.69.0
  run:
    - boost-cpp >=1.69.0
```
Do I need to build against multiple boost versions? Or fix the boost version? Not sure what to do here.

ping @conda-forge/help-c-cpp and @scopatz

**Comment by hadim** [comment]
(the boost version my env uses is 1.70.0)

**Comment by hadim** [comment]
Fixed by #2
